reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -431,11 +479,12 @@ def generate_slices(map_3d_f, xy_plane, rots):
                 xy_plane[0] is x coordinate
                 xy_plane[1] is y coordinate
                 xy_plane[2] is z coordinate, which is all zero
-        n_pix : int
-            Number of pixels along one edge of the plane.
         rots : arr
             Array describing rotations.
             Shape (n_rotations, n_pix**2, 3)
+        z_offset : float
+            Symmetrical z-depth given to the xy_plane before rotating.
+            0 < z_offset < 1
```

Comment:
Did you see my notes on the rotation.

I think we should remove this
```
        Shift the space into a centered position before rotating and
        revert shift after rotation. This preserves the bounds of the
        space.
```

Also, I don't follow the z_offset logic. do you want some sort of padding nearby?

Reply:
Yes exactly - I went into it here: https://github.com/compSPI/reconstructSPI/pull/44#discussion_r843337580 - the gist is that if we have a 2D slice in a 3D object, we can't meaningfully interpolate anything from it. Points "near" the slice are never "on" the slice because the slice is infinitely thin and we have to contend with float precision. In reality what this means is scipy throws an error when setting up the interpolation. 

The slice depth fixes this, and since the depth is much less than the width of a pixel, it shouldn't have any affect on the validity of our results.